api/src/test/java/com/microsoft/gctoolkit/time/DateTimeStampTest.java

```diff
@@ -15,19 +15,19 @@ public class DateTimeStampTest {
     private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSZ");
 
     @Test
-    public void getTimeStamp() {
+    void getTimeStamp() {
         DateTimeStamp dateTimeStamp = new DateTimeStamp(.586);
         assertEquals(.586, dateTimeStamp.getTimeStamp(), 0.0001);
 
         dateTimeStamp = new DateTimeStamp("2018-04-04T09:10:00.586-0100");
-        assertEquals(Double.NaN, dateTimeStamp.getTimeStamp(), 0.0001);
+        assertEquals(1.522836600586E12, dateTimeStamp.getTimeStamp(), 0.0001);
```

Comment:
The fact that you changed this test case means you changed the behavior of DateTimeStamp. This is a breaking change.

Reply:
I did change the behaviour but it isn't a breaking change AFAIK. The issues is what to do with logs that don't have a timestamp. Since there is (historically) a lot of behaviour that is based on the presence of a time stamp, not having a timestamp wasn't a viable option. To fix this the time in seconds from epoc is offered as a timestamp. About the only thing that affects is timestamp is consider to be the age of the JVM. However, given we have to deal with log fragements anyways, and date stamps don't give us the age of the JVM implies that using epoc as a starting point seems like a pragmatic solution.

The better way to manage IMO is to encapsulate *all* of the time calculations into DateTimeStamp. However, that would trigger another massive refactoring of all client code. I personally have no appetite for this at the moment.